frontend/src/reducers/strapi.js

```diff
@@ -0,0 +1,83 @@
+import Sponsor1 from "../assets/IDPH logo.PNG";
+import Sponsor2 from "../assets/24thDistrict Police.PNG";
+import Sponsor3 from "../assets/Citron Hygiene.jpg";
+import Sponsor4 from "../assets/evanston Police.jpg";
+import Sponsor5 from "../assets/first united methodist Church.png";
+import Sponsor6 from "../assets/FlatTop Grill.png";
+import Sponsor7 from "../assets/HomersIceCreamLogo200.jpg";
+import Sponsor8 from "../assets/Kiwanis Logo (1).jpg";
+import Sponsor9 from "../assets/Logo2.jpg";
+import Sponsor10 from "../assets/Roger's Park Library.PNG";
+import Sponsor11 from "../assets/uic logo2.png";
+import Sponsor12 from "../assets/Walmart_logo.svg.png";
+
+const initialState = {
+  email: "founder@ymim.org",
+  phone: "773.941.1200",
+  taxId: "82-1904373",
+  donateLink: "https://www.paypal.com/",
+  token: "",
+  facebook: "https://www.facebook.com/theymim/",
+  instagram: "https://www.instagram.com/theyoungmasterbuilders/",
+  twitter: "https://twitter.com/YMIMtweets",
+  video1: "https://www.youtube.com/embed/PK9ESRMGq74",
+  video2: "https://www.youtube.com/embed/jdsqht1m1rE",
+  video3: "https://www.youtube.com/embed/GwXt3tL6FqY",
+  sponsor1: Sponsor1,
+  sponsor2: Sponsor2,
+  sponsor3: Sponsor3,
+  sponsor4: Sponsor4,
+  sponsor5: Sponsor5,
+  sponsor6: Sponsor6,
+  sponsor7: Sponsor7,
+  sponsor8: Sponsor8,
+  sponsor9: Sponsor9,
+  sponsor10: Sponsor10,
+  sponsor11: Sponsor11,
+  sponsor12: Sponsor12
+};
+
+export default function getStrapi(state = initialState, action) {
+  switch (action.type) {
+    case "GET_EMAIL":
+      return { ...state, email: action.response.data[0].email };
```

Comment:
I do not think we are not going to be able to do `.data[0]`

I believe we are always going to want to take the last one for all of these? (most recently created) 

so maybe grab [last](https://lodash.com/docs/4.17.15#last) from lodash so we can do 

```
 let lastEmail = last(action.response.data)'
 if (lastEmail) {
   return { ...state, email: lastEmail.email };
}
```